airflow/dag_processing/processor.py

```diff
@@ -642,11 +654,14 @@ def execute_callbacks(
 
     @provide_session
     def _execute_dag_callbacks(self, dagbag: DagBag, request: DagCallbackRequest, session: Session):
-        dag = dagbag.dags[request.dag_id]
-        dag_run = dag.get_dagrun(run_id=request.run_id, session=session)
-        dag.handle_callback(
-            dagrun=dag_run, success=not request.is_failure_callback, reason=request.msg, session=session
-        )
+        try:
```

Comment:
Only needed to avoid errors for callbacks. I did not want to complicate it passing the "right" callbacks so "fake" callbacks I am passing now generate a lot of exceptions as dags/dag runs are not found,